### PR TITLE
Add exposure history legend

### PR DESCRIPTION
**Pull request by johnschoeman**
Why:
We would like to provide the user a way to view a legend of what the
indicators on the exposure history screen mean.

This commit:
Introduces a legend modal to the exposure history screen.

Co-Authored-By: devin jameson <devin@thoughtbot.com>

Screen Shot 2020-06-30 at 4 24 05 PM

### CODE REVIEW

File: app/styles/iconography.ts [hunk 1]
```diff
@@ -1,7 +1,17 @@
...
+import { ViewStyle, TextStyle } from 'react-native';
 
 import * as Colors from './colors';
 import * as Spacing from './spacing';
+import * as Typography from './typography';
+
+export const smallIcon: ViewStyle = {
+  height: 30,
+  width: 30,
+  borderRadius: 100,
```
Comment (jeffstolz) [inline app/styles/iconography.ts:10]: perhaps set this to 100%?

**Review by jeffstolz — APPROVED** [review]
Approved, I just recommend checking this on the iPhone 8 before merging to make sure those absolute position values don't need to be adjusted

**Comment by johnschoeman** [comment]
> Approved, I just recommend checking this on the iPhone 8 before merging to make sure those absolute position values don't need to be adjusted

by iPhone 8 do you mean iPhone11? if so then, yup we did so, just didnt post a pic.

File: app/styles/iconography.ts (hunk 1 above)
Comment (johnschoeman) [reply app/styles/iconography.ts:10]: sadly border radius doesnt accept percentages.

**Pull request merged**